assets/prefabs/PearSeed.prefab

```diff
@@ -0,0 +1,31 @@
+{
+  "parent": "engine:iconItem",
+  "DisplayName": {
+    "name": "Pear Seed"
+  },
+  "Item": {
+    "icon": "SimpleFarming:SimpleFarming#PearSeed",

```

Comment:
This doesn't work as the `#` indicates looking inside a texture atlas (one image with a bunch of little images in it). Should be `SimpleFarming:PearSeed` to match your texture file

Please be sure to run the game and test that your content works :-)